seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/sql/zeta/ZetaSQLType.java

```diff
@@ -85,6 +86,24 @@ public class ZetaSQLType {
     public static final String TIME = "TIME";
     public static final String BOOLEAN = "BOOLEAN";
 
+    public static final List<SqlType> DATETIME_CAST_TYPES =
```

Comment:
Thanks @liuwei178 . Can we refactor it and move the cast-related logic and validation rules into separate classes?